NEWS.md

```diff
@@ -164,6 +164,10 @@ Standard library changes
 * `(+)(::UniformScaling)` is now defined, making `+I` a valid unary operation ([#36784]).
 * Instances of `UniformScaling` are no longer `isequal` to matrices. Previous
   behaviour violated the rule that `isequal(x, y)` implies `hash(x) == hash(y)`.
+* Transposing `*Triangular` matrices now returns matrices of the opposite triangular type, consistently
```

Comment:
```suggestion
* Transposing `*Triangular` matrices now returns matrices of the opposite triangular type, consistent
```

It is an adverbial use, but `consistent` sounds better than `consistently` for me here.

(I'm technically a native speaker, but I would not depend on my intuitions.)

Reply:
I believe the original is right, and also sounds better.